src/components/GordonDialogBox/index.jsx

```diff
@@ -41,6 +41,7 @@ const GordonDialogBox = ({
       <DialogTitle
         className={titleClass ? titleClass : styles.gc360_gordondialogbox_title}
         id="alert-dialog-title"
+        sx={severity ? { bgcolor: `${severity}.main`, pb: '0' } : { bgcolor: 'primary.main' }}
```

Comment:
```suggestion
        sx={{ bgcolor: `${severity ?? 'primary'}.main`, pb: '0' }}
```
a very nice change! but wouldn't this be the same as ^
I could be wrong

Reply:
all g